cmd/exocored/root.go

```diff
@@ -350,3 +349,24 @@ func initTendermintConfig() *tmcfg.Config {
 
 	return cfg
 }
+
+// getChainID loads the chainID from the flag or the config.toml file, with the
+// former taking precedence over the latter.
+func getChainID(appOpts servertypes.AppOptions, home string) string {
```

Comment:
I made this function since to avoid repeated code. It is called during `exocored start` as well as `exocored export`.